### PR TITLE
use correct name of inflixdb service in openshift

### PR DESCRIPTION
**Pull request by dejanb**

### CODE REVIEW

File: example/src/main/config/hono-adapter-mqtt-vertx-config.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@ hono:
     reporter:
       graphite:
         active: true
-        host: influxdb.hono
+        host: influxdb
```
Comment (sophokles73) [inline example/src/main/config/hono-adapter-mqtt-vertx-config.yml]: this file is not just used when deploying to Openshift. Without the `.hono` suffix it will not work in Docker Swarm with certain DNS configuration of the host system. Is there a problem you are experiencing when deploying to Openshift using the original names? For me it works out-of-the-box without any problems ...

Comment (dejanb) [reply example/src/main/config/hono-adapter-mqtt-vertx-config.yml]: @sophokles73 Without the change I get a lot of these

```
09:45:22.894 [metrics-graphite-reporter-1-thread-1] WARN  c.c.m.graphite.GraphiteReporter - Unable to report to Graphite
  | java.net.UnknownHostException: influxdb.hono
  | at com.codahale.metrics.graphite.Graphite.connect(Graphite.java:122)
  | at com.codahale.metrics.graphite.GraphiteReporter.report(GraphiteReporter.java:240)
  | at com.codahale.metrics.ScheduledReporter.report(ScheduledReporter.java:251)
  | at com.codahale.metrics.ScheduledReporter$1.run(ScheduledReporter.java:174)
  | at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
  | at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
  | at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
  | at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
  | at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
  | at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
  | at java.lang.Thread.run(Thread.java:748)
 ```

Maybe renaming the service to hono-influxdb is a better approach?

Comment (dejanb) [reply example/src/main/config/hono-adapter-mqtt-vertx-config.yml]: This is in minishift environment

Comment (sophokles73) [reply example/src/main/config/hono-adapter-mqtt-vertx-config.yml]: let me try again locally ... I will come back to you later.

Comment (sophokles73) [reply example/src/main/config/hono-adapter-mqtt-vertx-config.yml]: BTW which version of minishift are you using, @dejanb ?

**Comment by dejanb** [comment]
@sophokles73 minishift upgrade did the trick. Closing this now. Thanks!
